Path-Check/gaen-mobile
# Feature/firebase crashlytics android

#### Description:

Install and activate Firebase Crashlytics for Android

#### Linked issues:



#### Screenshots:



#### How to test:

Insert code into MainActivity.java to make the app crash.  Run the app.  After it crashes, remove the crashing code and run again.  Go to Firebase console, project Pathcheck-GAEN-dev, app pathcheck-gaen-dev.  Click on Crashlytics on left and see the crash
